mailing_list/models.py

```diff
@@ -168,6 +168,17 @@ def get_or_create_or_delete_mailing_lists_for_canvas_course_id(self, canvas_cour
         return result
 
 
+class CourseSettings(models.Model):
+    canvas_course_id = models.IntegerField()
+    always_mail_staff = models.NullBooleanField(null=True, default=True)
+    modified_by = models.CharField(null=True, max_length=32)
+    date_created = models.DateTimeField(null=True, default=timezone.now)
```

Comment:
We could also use `auto_now_add=True` if we don't ever expect a user to be manually setting the `date_created` field.